File: okex-java-sdk-api-v5/src/main/java/com/okex/open/api/service/affiliate/impl/AffiliateAPI.java
package com.okex.open.api.service.affiliate.impl;


import com.alibaba.fastjson.JSONObject;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;
import retrofit2.http.Query;

public interface AffiliateAPI {



    @GET("/api/v5/affiliate/invitee/detail")
    Call<JSONObject> getInviteeDetail(@Query("uid")String uid);

    @GET("/api/v5/users/partner/if-rebate")
    Call<JSONObject> getPartner(@Query("apiKey")String apiKey);
}

File: okex-php-sdk-api-v5/src/AffiliateApi.php
<?php


namespace okv5;


class AffiliateApi extends Utils
{
    public function detail($uid){
        $params = [
            'uid' => $uid,
        ];

        return $this->request('/api/v5/affiliate/invitee/detail', $params, 'GET');
    }


}
